sktime/proba/normal.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, mu, sigma, index=None, columns=None):
         # and broadcast of parameters.
         # move this functionality to the base class
         # 0.19.0?
-        self._mu, self._sigma = self._get_bc_params()
+        self._mu, self._sigma = self._get_bc_params(*(self.mu, self.sigma))
```

Comment:
This should be the same as `= self._get_bc_params(self.mu, self.sigma)`, no?

Reply:
Yes, changed in 9473e92.